lib/components/artist/articles/index.js

```diff
@@ -41,3 +39,16 @@ const styles = StyleSheet.create({
     fontSize: 20,
   }
 });
+
+export default Relay.createContainer(Articles, {
+  fragments: {
+    artist: () => Relay.QL`
+      fragment on Artist {
+        name
```

Comment:
Is `name` still really used here? If not, I think you can do the same type of collection query we used on the `shows/short_list` and `shows/long_list` with `@relay(plural: true)`, that way this component only needs to receive the articles props and no artist.